setup.py

```diff
@@ -42,6 +42,7 @@
     "python-dateutil>=2.1,<2.8.1",
     "python-json-logger",
     "requests",
+    "requests_toolbelt",
```

Comment:
As far as I am able to tell, this is the easiest way to parse a multipart request with just its contents (since we don't have a `Request` from Lambda)